bin/YAMLTestSuite.pm

```diff
@@ -0,0 +1,99 @@
+package YAMLTestSuite;
+use v5.18;
+
+use utf8;
+use autodie qw(open close);
+use Encode;
+use MIME::Base64;
+use YAML::PP;
+
+my $ypp = YAML::PP->new;
+
+sub new {
+  my $class = shift;
```

Comment:
For perl, can we please use 4 spaces indentation? I find 2 spaces hard to read.

Reply:
Done